src/scripts/components/header/header.tsx

```diff
@@ -25,6 +25,7 @@ const Header: FunctionComponent<Props> = ({
       link={backLink}
     />
     <h1 className={styles.title}>{title}</h1>
+    {!children && <div className={styles.empty}></div>}
```

Comment:
Probably better to wrap the children in a div and give the wrapper div these styles. This way we don't need the empty div and the showcase selector does not need any styles specific to the header